pkg/cmd/pulumi-language-java/main.go

```diff
@@ -314,9 +338,42 @@ func (host *javaLanguageHost) Run(_ context.Context, req *pulumirpc.RunRequest)
 	var stdoutBuf bytes.Buffer
 	var stderrBuf bytes.Buffer
 
-	cmd.Stdout = &stdoutBuf
+	pr, pw := io.Pipe()
+
+	cmd.Stdout = pw
```

Comment:
A comment might be helpful here as it took me a minute to work out what's going on. As I understand it, we want to keep sending the command's output to stdout, but we also need to process it to see when the debugger is ready -- is that right?